app/ui/src/app/platform/types/integration/integration.models.ts

```diff
@@ -91,6 +92,13 @@ export interface Flow extends WithId {
   connections: Array<Connection>;
 }
 
+export const PRIMARY = 'PRIMARY';
+export const ALTERNATE = 'ALTERNATE';
+
+export type FlowType =
+  | 'PRIMARY'
+  | 'ALTERNATE';
```

Comment:
Might want to use the `const` variables from above instead?

Reply:
that was my 1st thought, too. but for some reason TypeScript won't let me use the const values there 🤷‍♂

Reply:
Would [string enum](https://www.typescriptlang.org/docs/handbook/enums.html) be a better choice here?